testnets/pockettestnet/chain.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "pockettestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Beta TestNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket-beta",
+  "bech32_prefix": "pokt@",
+  "daemon_name": "poktrolld",
+  "node_home": "$HOME/.poktroll",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "upokt",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.0025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "upokt"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/pokt-network/poktroll",
+    "recommended_version": "v0.0.11-rc",
+    "compatible_versions": ["v0.0.11-rc"],
+    "binaries": {
+      "linux/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_linux_amd64.tar.gz",
+      "linux/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_darwin_amd64.tar.gz",
+      "darwin/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_darwin_arm64.tar.gz"
+    },
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.10"
+    },
+    "genesis": {
+      "name": "v0.0.11-rc",
+      "genesis_url": "https://github.com/pokt-network/pocket-network-genesis/blob/master/shannon/testnet/genesis.json"
+    },
+    "versions": [
```

Comment:
`"versions"` was removed from `codebase` and has been relocated to a separate file `versions.json`